packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -416,6 +430,28 @@ export const start = async (zcf, privateArgs, baggage) => {
     appendToStoredArray(deposits, amount.brand, harden({ seat, amount, goal }));
   };
 
+  const sendToReserve = keyword => {
+    const { reservePublicFacet } = zcf.getTerms();
+
+    const amount = reserveSeat.getCurrentAllocation()[keyword];
+    if (!amount || AmountMath.isEmpty(amount)) {
+      return;
+    }
+
+    const invitation = E(reservePublicFacet).makeAddCollateralInvitation();
+    // don't wait for a response
+    void E.when(invitation, invite => {
```

Comment:
what if the `invitation` promise or the handler fails?

I notice the handler explicitly ignores failure of `offerTo`. Why is that safe?

Reply:
If either fails, the assets will stay on the reserveSeat and will be sent the next time this function is called. I don't expect that to happen, and when it does, retrying immediately seems unlikely to be the answer. 

It's safe because it's self-healing, and there doesn't seem to be anything useful to do with a failure.